doc/design/session.md

```diff
@@ -0,0 +1,62 @@
+# Design Doc: Session
+
+## Abstract
+
+This design doc proposes to have an object called *Session* which
+encapsulates the environment in which the computation graph is
+executed.
+
+## Background
+
+A computation graph is executed in an environment which contains the
+[scope](./scope.md) and other states. PaddlePaddle used to only have
```

Comment:
"used to only have" => "used only to have" or "used to have only". Indeed, here we are describing the current statues, so it should be

> The current design has an implicit session ...

Reply:
Thanks! Done.